packages/docusaurus-plugin-content-blog/src/plugin-content-blog.d.ts

```diff
@@ -451,6 +462,52 @@ declare module '@docusaurus/plugin-content-blog' {
     content: string;
   };
 
+  export type BlogFeedItem = {
```

Comment:
wonder if it's useful to have our own type there if we only copy it from the original lib? 🤔 

I guess it's only worth it if we create our own API surface over it (eventually not exposing everything), otherwise just aliasing lib types may be enough?

I just want to avoid possible lib breaking changes being un-noticed, but maybe I'm a bit too careful and it's not worth it? 🤷‍♂️

Reply:
I'm happy with merely aliasing it - have switched to that instead.  I think you're right; it's probably safer with regards breaking changes